modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/DesignPacksProjectConfigurator.java

```diff
@@ -172,13 +200,8 @@ private Zip _addTaskBuildDesignPack(
 
 		zip.dependsOn(buildCSSTask);
 
-		zip.from(new File(buildThemeTask.getOutputDir(), "/css/"));
```

Comment:
we can not use this way, It will cause the the dsign pack zip contain absoulute path for css files. like "liferay-workspace/desgin-pack/fire1/css/.....". But it will correct when I use task.into to copy css files.